### PR TITLE
Add cane/wheelchair crate

### PR DESCRIPTION
**Pull request by Kezu02**
## About the PR
I added two crates: the blind support crate and wheelchair crate. Both cost 300 spesos.

## Why / Balance
Blind/wheelchair players will inevitably get their things lost/taken. There wasn't any way to get more.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: NanoTrasen has begun producing spare canes and wheelchairs for our disabled crew. Order them from Cargo!

### CODE REVIEW

**Comment by brainfood1183** [comment]
we likely could do with a wheelchair spawner as well so wheelchair could be mapped in medical

**Comment by Aisu9** [comment]
what about crafting them in lathes

File: Resources/Prototypes/Catalog/Cargo/cargo_emergency.yml [hunk 1]
```diff
@@ -77,3 +77,23 @@
...
+  id: EmergencyCane
+  icon:
+    sprite: Objects/Weapons/Melee/white_cane.rsi
+    state: icon
+  product: CrateBlindSupport
+  cost: 300
+  category: Emergency
+  group: market
+
+  - type: cargoProduct
```
Comment (lzk228) [inline Resources/Prototypes/Catalog/Cargo/cargo_emergency.yml:91]:
```suggestion
- type: cargoProduct
```

**Comment by QuietlyWhisper** [comment]
resolves #19753

**Comment by mirrorcult** [comment]
please do not use github webedits to make PRs, and include screenshots of this working as expected in game

**Review by mirrorcult — CHANGES_REQUESTED** [review]
above

**Comment by LankLTE** [comment]
Superseded by #19811 . Feel free to re-open if you wish.

**Pull request closed without merging**